man/systemd-swap.8

```diff
@@ -1,9 +1,9 @@
-.TH systemd-swap 8 "JUNE 2020" "systemd-swap 4.2" systemd-swap
+.TH systemd-swap 8 "JULY 2020" "systemd-swap 5.0" systemd-swap
```

Comment:
Actually, I was planning on having this as 4.4 following https://semver.org/
I'll do a write up of future plans and dreams in an issue later.

Reply:
Oh, alright. I was going to ask, but assumed the changes might be too much to bump just the minor version. Especially the conf file handling is different now and might not always produce exactly the same results compared to the Bash version of systemd-swap.

Reply:
I agree that bumping to 5.0 would be valid, but I fear that we'd bump to 6.0 in under a year, and that feels like moving a bit too far. (Essentially the plan is to replace swapfc with [swapspace](https://github.com/Tookmund/Swapspace) for 5.0)

Reply:
We could implement both changes at the same time (have master be on python and only release 5.0 when swapspace is merged), but then the python rewrite gets delay a lot

Reply:
Then I'd say adjusting only the minor version is fine. The changes are probably unlikely to break much unless some people got crazy with their `swap.conf` file.

Reply:
Great, I'll start looking and running the python code soon (no promises on time, but since it's such a large patch it'll probably take a week)

Reply:
Thanks. I've changed the version to 4.4. It doesn't matter if it takes a week or longer, as long as we make sure that the rewrite didn't break anything.